### PR TITLE
vegman: tech: add FRU updating module

### PR DESCRIPTION
**Pull request by alatarum**
This brings implementation for service function to update particular FRU
fields. To do this the script restarts FruDevice daemon in special mode,
where number of parameters become writable.

Signed-off-by: Andrei Kartashev <a.kartashev@yadro.com>

### CODE REVIEW

**Comment by nest1ing** [comment]
Is it really need to be a part of the CLI project?
As I understood, this tool is very specific. It will be used by our manufacture guys only

**Comment by nest1ing** [comment]
Oops, sorry. Missclick.

File: commands/tech.vegman [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    fi
+  done
+
+  echo "Parameters to be set in FRU:"
+  echo "    CHASSIS_PART_NUMBER:   ${fru_chassis_pnum}"
+  echo "    PRODUCT_PART_NUMBER:   ${fru_product_pnum}"
+  echo "    PRODUCT_PRODUCT_NAME:  ${fru_product_name}"
+  echo "    PRODUCT_SERIAL_NUMBER: ${fru_product_serial}"
+  if confirm; then
+    set_fru_field "CHASSIS_PART_NUMBER" "${fru_chassis_pnum}" || return 1
```
Comment (artemsen) [inline commands/tech.vegman:149]: `|| return 1` is not necessary, you have already set `-e` for entire script.

Comment (alatarum) [reply commands/tech.vegman:149]: IMHO, code should not rely on command line options.

Comment (AlexanderAmelkin) [reply commands/tech.vegman:149]: `set -e` is not a command line option. That's what is set in the script itself, and that's a reliable feature.

**Pull request closed without merging**